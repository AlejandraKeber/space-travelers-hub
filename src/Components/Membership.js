import React from 'react';

export default function Membership(props) {
  const { reserved } = props;
  let membership;
  if (reserved) {
    membership = <td className="status active">Active member</td>;
  } else {
    membership = <td className="status">Not a member</td>;
  }
  return (
    membership
  );
}
