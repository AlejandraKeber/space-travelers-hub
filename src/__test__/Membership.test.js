import React from 'react';
import renderer from 'react-test-renderer';
import Membership from '../Components/Membership';

describe('Membership component', () => {
  test('Test Membership render correctly', () => {
    const tree = renderer.create(<Membership />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
