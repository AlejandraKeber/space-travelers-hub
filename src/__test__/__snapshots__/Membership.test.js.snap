// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Membership component Test Membership render correctly 1`] = `
<td
  className="status"
>
  <button
    type="button"
  >
    Not a member
  </button>
</td>
`;
